### PR TITLE
Change unity3d download page

### LINKED ISSUE #415: Cannot install unity3d in umake --version 16.11.1
I want to install unity3d in ubuntu PC by umake. But I couldn't do it. This is Error Message.
```
$ umake games unity3d
Choose installation path: /home/takaken/.local/share/umake/games/unity3d
ERROR: Download page changed its syntax or is not parsable (url missing)
```

This is my umake version.
```
$ umake --version
16.11.1
```

### PR DESCRIPTION
**Pull request by LyzardKing**
@didrocks This still needs the tests to be fixed.
It fixes #415. The page has been changed, and the posts now link to a separate page for each version download. It should work, but I don't have much time to properly test this at the moment.

### CODE REVIEW

**Comment by didrocks** [comment]
Do you have time to work on adapt the medium test for this? (I'm going to run the large one as we discussed)

**Comment by LyzardKing** [comment]
I had already changed the medium tests, I forgot to upload the changes.
I get the following error:

ERROR: https://forum.unity3d.com/threads/unity-on-linux-release-notes-and-known-issues.350256 couldn't finish download: HTTPSConnectionPool(host='forum.unity3d.com', port=443): Max retries exceeded with url: /threads/unity-on-linux-release-notes-and-known-issues.350256 (Caused by <class 'ConnectionRefusedError'>: [Errno 111] Connection refused)
ERROR: An error occurred while downloading https://forum.unity3d.com/threads/unity-on-linux-release-notes-and-known-issues.350256: HTTPSConnectionPool(host='forum.unity3d.com', port=443): Max retries exceeded with url: /threads/unity-on-linux-release-notes-and-known-issues.350256 (Caused by <class 'ConnectionRefusedError'>: [Errno 111] Connection refused)

File: tests/medium/__init__.py [hunk 1]
```diff
@@ -31,7 +31,8 @@ class ContainerTests(LoggedTestCase):
     """Container-based tests utilities"""
 
     DOCKER_USER = "user"
-    DOCKER_TESTIMAGE = "didrocks/docker-umake-manual"
+    #DOCKER_TESTIMAGE = "didrocks/docker-umake-manual"
+    DOCKER_TESTIMAGE = "7e2311e7a154"
```
Comment (didrocks) [inline tests/medium/__init__.py]: argh, you did push your local change…

File: umake/frameworks/games.py [hunk 2]
```diff
@@ -112,26 +112,26 @@ def __init__(self, category):
...
-                                       "threads/unity-on-linux-release-notes-and-known-issues.350256/",
+                                       "threads/unity-on-linux-release-notes-and-known-issues.350256",
                          match_last_link=True,
                          checksum_type=ChecksumType.sha1,
                          dir_to_decompress_in_tarball='unity-editor*',
                          desktop_filename="unity3d-editor.desktop",
                          required_files_path=[os.path.join("Editor", "Unity")],
                          # we need root access for chrome sandbox setUID
-                         need_root_access=True,
+                         need_root_access=False)
```
Comment (didrocks) [inline umake/frameworks/games.py]: who this change? the chrome sandbox setUID isn't needed anymore?

File: umake/frameworks/games.py [hunk 3]
```diff
@@ -112,26 +112,26 @@ def __init__(self, category):
...
-                             "gconf-service", "lib32gcc1", "lib32stdc++6", "libasound2", "libcairo2",
-                             "libcap2", "libcups2", "libfontconfig1", "libfreetype6", "libgconf-2-4",
-                             "libgdk-pixbuf2.0-0", "libglu1-mesa", "libgtk2.0-0",
-                             "libgl1-mesa-glx | libgl1-mesa-glx-lts-utopic |\
-                              libgl1-mesa-glx-lts-vivid | libgl1-mesa-glx-lts-wily",
-                             "libnspr4", "libnss3", "libpango1.0-0", "libpq5", "libxcomposite1",
-                             "libxcursor1", "libxdamage1", "libxext6", "libxfixes3", "libxi6",
-                             "libxrandr2", "libxrender1", "libxtst6",
-                             "monodevelop"])  # monodevelop is for mono deps, temporary
+                         #~ packages_requirements=[
```
Comment (didrocks) [inline umake/frameworks/games.py]: same, those packages aren't needed?

**Comment by didrocks** [comment]
(if the above was just for test changes, do you mind ammending your commit and push -f? so that they don't appear in history)

**Comment by LyzardKing** [comment]
Done. the previous error was because I had not switched to 443 in the test.
I'm still testing...

File: umake/frameworks/games.py [hunk 4]
```diff
@@ -111,42 +111,47 @@ class Unity3D(umake.frameworks.baseinstaller.BaseInstaller):
...
+                         download_page="https://forum.unity3d.com/" +
+                                       "threads/unity-on-linux-release-notes-and-known-issues.350256",
                          match_last_link=True,
                          checksum_type=ChecksumType.sha1,
                          dir_to_decompress_in_tarball='unity-editor*',
                          desktop_filename="unity3d-editor.desktop",
                          required_files_path=[os.path.join("Editor", "Unity")],
                          # we need root access for chrome sandbox setUID
-                         need_root_access=True,
+                         need_root_access=False,
```
Comment (didrocks) [inline umake/frameworks/games.py]: It's still set to false, why is it?

File: umake/frameworks/games.py [hunk 5]
```diff
@@ -111,42 +111,47 @@ class Unity3D(umake.frameworks.baseinstaller.BaseInstaller):
...
-                             "gconf-service", "lib32gcc1", "lib32stdc++6", "libasound2", "libcairo2",
-                             "libcap2", "libcups2", "libfontconfig1", "libfreetype6", "libgconf-2-4",
-                             "libgdk-pixbuf2.0-0", "libglu1-mesa", "libgtk2.0-0",
-                             "libgl1-mesa-glx | libgl1-mesa-glx-lts-utopic |\
-                              libgl1-mesa-glx-lts-vivid | libgl1-mesa-glx-lts-wily",
-                             "libnspr4", "libnss3", "libpango1.0-0", "libpq5", "libxcomposite1",
-                             "libxcursor1", "libxdamage1", "libxext6", "libxfixes3", "libxi6",
-                             "libxrandr2", "libxrender1", "libxtst6",
-                             "monodevelop"])  # monodevelop is for mono deps, temporary
+                              "gconf-service", "lib32gcc1", "lib32stdc++6", "libasound2", "libcairo2",
```
Comment (didrocks) [inline umake/frameworks/games.py]: you added an extra space, mind fixing this?

File: umake/frameworks/games.py (hunk 4 above)
Comment (LyzardKing) [inline umake/frameworks/games.py]: I changed that to run a quick test when I first changed the url...I forgot to change it.
Changed now.

File: umake/frameworks/games.py (hunk 5 above)
Comment (LyzardKing) [inline umake/frameworks/games.py]: Sure

**Comment by didrocks** [comment]
All changes looks good! Let's do the small function refactoring later on as we discussed.

**Pull request merged**